Netable/Example/Requests/GraphQL/GetAllPostsQuery.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  GetAllPostsQuery.swift
+//  Example
+//
+//  Created by Amy Oulton on 2022-12-08.
+//  Copyright © 2022 Steamclock Software. All rights reserved.
+//
+
+import Foundation
+import Netable
+
+struct GetAllPostsQuery: GraphQLRequest {
+    typealias Parameters = Empty
+    typealias RawResource = SmartUnwrap<[Post]>
```

Comment:
I opted to change this one to be a SmartUnwrap because they were both using finalize!